self_positioning:
  rate: 50
  agents_no: 5
  target_sub: /target/pose
  trajectory_pub: /uavs/multiagent_command/pose
  formation_sub: /uavs/formation_state
  angle_sub: /uavs/formation/update
  formation_geometry:
    - pos:
        x: -2.0
        y: 0.0
        z: 2.0
      rot:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
    - pos:
        x: -2.0
        y: 1.0
        z: 2.0
      rot:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
    - pos:
        x: -2.0
        y: -1.0
        z: 2.0
      rot:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
    - pos:
        x: -2.0
        y: 2.0
        z: 2.0
      rot:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
    - pos:
        x: -2.0
        y: -2.0
        z: 2.0
      rot:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
